docs/docs/spec.md

```diff
@@ -227,7 +227,9 @@ exceptions in the subsequent sections:
     > This excludes the receiver parameter but includes variadic parameters.
     > Specifically, you can add `@Nullable` before the `...` token to indicate
     > that a variadic method accepts `null` arrays: `void foo(String @Nullable
-    > ... strings)`.
+    > ... strings)`. This syntax follows the syntax for normal arrays: A method
+    > that accepts a `null` array is written as `void foo(String @Nullable []
+    > strings)`.
```

Comment:
I added some text based on that. It sounds like you might also be suggesting that I remove the comment under "array component type," since we now cover that here?

We can pick this up in https://github.com/jspecify/jspecify/pull/674.